demo/src/main/java/com/example/demo/model/DTO/ContactMethodDTO.java

```diff
@@ -0,0 +1,35 @@
+package com.example.demo.model.DTO;
+
+import com.example.demo.model.ContactMethodType;
+
+import com.example.demo.model.ValidContactMethod;
+import lombok.Builder;
+import lombok.Data;
+
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+
+@Data
```

Comment:
Instead of @Data annotation I would use specific setter, getter and constructor annotations.